src/lib/multicall/Multicall.ts

```diff
@@ -214,6 +218,7 @@ export class Multicall {
           isInMainThread: !isWebWorker,
           requestType,
           rpcProvider,
+          isLargeAccount,
```

Comment:
i think it could be added to useConfigureMetrics hooks and settled as GlobalMetricData